### DISCUSSION TITLE
Possible to allow `quote` and `escape` options in `fixlengths` command?

### DISCUSSION THREAD

**Discussion by rad-pat** [dathere/qsv#2102] [Ideas]
We process CSVs from customers that come in all shapes and sizes, running through `fixlengths` is great to ensure that the CSVs are correct. However, they also don't always come quoted and escaped using double-quotes. Is it possible to add those options to the `fixlengths` command since the file itself has to be passed? If the lengths are incorrect, then `input` will complain about this when loading.

**Comment by jqnatividad** [comment]
Hi @rad-pat , this is a great idea. Adding it to the backlog.

**Reply by jqnatividad** [reply]
Implemented in https://github.com/jqnatividad/qsv/issues/2103

Please let me know how it works for you...

**Reply by rad-pat** [reply]
Wow, amazing. I'll let you know how I get on when I get chance to have a look, thanks!

**Comment by jqnatividad** [comment]
Closing discussion now that the request has been implemented.
